src/notional.ts

```diff
@@ -226,6 +219,8 @@ export function handleUpdateETHRate(event: UpdateETHRate): void {
 
   log.debug('Updated ethExchangeRate variables for entity {}', [ethExchangeRate.id]);
   ethExchangeRate.save();
+
+  updateEthExchangeRateHistoricalData(ethExchangeRate, event);
```

Comment:
@ddallaire Putting this function call here won't do what you think it does, this `handleUpdateETHRate` will only be called rarely when the ETHRate configuration is updated by governance.

To get hourly updated exchange rate data you will need to register your function inside the subgraph template as a block handler:
https://thegraph.com/docs/en/developer/create-subgraph-hosted/#block-handlers